docs/source/common/weights_loading.rst

```diff
@@ -41,83 +43,70 @@ To change the checkpoint path pass in:
     # saves checkpoints to '/your/path/to/save/checkpoints' at every epoch end
     trainer = Trainer(default_root_dir="/your/path/to/save/checkpoints")
 
-You can customize the checkpointing behavior to monitor any quantity of your training or validation steps. For example, if you want to update your checkpoints based on your validation loss:
-
-1. Calculate any metric or other quantity you wish to monitor, such as validation loss.
-2. Log the quantity using :func:`~~pytorch_lightning.core.lightning.LightningModule.log` method, with a key such as `val_loss`.
-3. Initializing the :class:`~pytorch_lightning.callbacks.ModelCheckpoint` callback, and set `monitor` to be the key of your quantity.
-4. Pass the callback to the `callbacks` :class:`~pytorch_lightning.trainer.Trainer` flag.
-
-.. testcode::
-
-    from pytorch_lightning.callbacks import ModelCheckpoint
-
-
-    class LitAutoEncoder(LightningModule):
-        def validation_step(self, batch, batch_idx):
-            x, y = batch
-            y_hat = self.backbone(x)
+You can retrieve the checkpoint after training by calling:
 
-            # 1. calculate loss
-            loss = F.cross_entropy(y_hat, y)
+.. code-block:: python
 
-            # 2. log `val_loss`
-            self.log("val_loss", loss)
+        checkpoint_callback = ModelCheckpoint(dirpath="my/path/")
+        trainer = Trainer(callbacks=[checkpoint_callback])
+        trainer.fit(model)
+        checkpoint_callback.best_model_path
```

Comment:
does this work? is `best_model_path` defined if there's no `monitor` set in the ModelCheckpoint callback?

Reply:
[yep](https://github.com/PyTorchLightning/pytorch-lightning/blob/3fcfd0214cc95fa7de49f6404421b5c6cd113c57/pytorch_lightning/callbacks/model_checkpoint.py#L677). I guess its the best 1/1, though the name is even more confusing when `save_top_k = -1` and `monitor = None`. maybe we could better rename this in the future, sth like `self.latest_model_path` ?

Reply:
updated!